docs/application_packages.md

```diff
@@ -28,7 +28,7 @@ appropriate values (make sure this is all on one line):
 
     $ mvn archetype:generate \
      -DarchetypeGroupId=org.apache.apex \
-     -DarchetypeArtifactId=apex-app-archetype -DarchetypeVersion=3.4.0 \
+     -DarchetypeArtifactId=apex-app-archetype -DarchetypeVersion=RELEASE \
```

Comment:
I think it is better to use RELEASE in the example as that is what users almost always want when creating a new project. I added a line above to address this.